web/src/pages/ChatAgent/components/SourcesPanel.css

```
@@ -0,0 +1,68 @@
+  transition: height 260ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.source-deck-card {
+  transition:
+    transform 260ms cubic-bezier(0.16, 1, 0.3, 1),
+    opacity 260ms cubic-bezier(0.16, 1, 0.3, 1),
+    box-shadow 120ms cubic-bezier(0.16, 1, 0.3, 1);
+  animation: sourceDeckCardIn 280ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+@keyframes sourceDeckCardIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "sourceDeckCardIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)